docs/poc2023q1/edge-scheduler.md

```diff
@@ -0,0 +1,65 @@
+### Edge scheduler
+The edge scheduler monitors the EdgePlacement, Location, and SyncTarget objects and maintains the results of matching.
+
+It starts from the kcp-scheduling-placement controller separated from kcp-dev/kcp, now being refactored towards the goal above.
```

Comment:
Perhaps we could say "forked" instead of "separated"

Reply:
Fixed in latest force-push.